### DISCUSSION TITLE
give me a more specific guide to the option of the @mdit/plugin-tab plugin

### DISCUSSION THREAD

**Discussion by namnguyen315** [mdit-plugins/mdit-plugins#48] [Q&A] [answered]
I don't understand how to add these options and how to configure them
![image](https://github.com/user-attachments/assets/93ee5f45-20e4-422d-a29c-cb31d88fe9bb)

**Comment by Mister-Hope** [comment, marked as answer]
These are accepting standard markdown-it renderer, which requires you to understand how markdown-it works, and that's not my job.

E.g.: See the official container plugin (written in js) https://github.com/markdown-it/markdown-it-container?tab=readme-ov-file, there is also no detailed examples.

You can get the default renderer at https://github.com/mdit-plugins/mdit-plugins/blob/a46e5898319b34a9f9558e50cb0094ebf8bf4b22/packages/tab/src/plugin.ts#L331-L389.

Test and snapshots are at https://github.com/mdit-plugins/mdit-plugins/blob/main/packages/tab/__tests__/tab.spec.ts  and https://github.com/mdit-plugins/mdit-plugins/blob/main/packages/tab/__tests__/__snapshots__/tab.spec.ts.snap

And finally, again:

> which requires you to understand how markdown-it works
